manifests/build.pp

```diff
@@ -0,0 +1,63 @@
+# Configure the system for module building
+#
+# Config for module building
+# --------------------------
+#
+# The module building requires the following file structure:
+#
+# ```
+# $module_build_root/
+#   bin/ # for simple module build script
+#   modules/ # module source files and compiled policies
+#   modules/tmp # repolicy tempfiles (created by scripts)
+# ```
+#
+# @param module_build_root See main class
+#
+# @api private
+#
+class selinux::build(
+  $module_build_root = $::selinux::module_build_root,
```

Comment:
please don't add `::` after the `$` and add a datatype

Reply:
crap, `::` is used all over the place. should we keep it here as well? Up to you.

Reply:
I considered and I preferred to be consistent so kept it.